web/index.html

```diff
@@ -7,5 +7,10 @@
     <h1 class="fullname">Robert Alexandru</h1>
     <h2 class="careerlvl">Junior Web Developer</h2>
     <h4 class="about-me">About me</h4>
+    <div class="message__wrapper">
+        <p class="message">Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque, animi dolorem illo in nihil dolore. Tempore est magni molestiae laudantium nulla libero voluptas, quas aut accusantium repudiandae neque debitis consequuntur?
```

Comment:
this sounds more like a `message__description` from the structure you've started